dev/tools/gen_defaults/generated/used_tokens.csv

```diff
@@ -1,5 +1,7 @@
-Versions used, v0_206
+Versions used, v2_3_5, v0_206
```

Comment:
curious what is still using v 0.206?

Reply:
Yeah, I think this is because my last commit change. I reverted the tokens in slider.json because there're a big change on `Slider` which might cause Scuba failures. I think probably we can create a separate PR for it:)